upi/aws/cloudformation/02_cluster_infra.yaml

```diff
@@ -1,36 +1,36 @@
 AWSTemplateFormatVersion: 2010-09-09
-Description: Template for Openshift Cluster UPI Network Elements (Route53 & LBs)
+Description: Template for OpenShift Cluster User-Provisioned Network Elements (Route53 & LBs)
```

Comment:
Can we just drop `UPI`?  I don't see a need to say "UPI", "User-Provisioned", or anything like that here.